src/System.CommandLine.Tests/SuggestDirectiveTests.cs

```diff
@@ -25,7 +25,7 @@ public SuggestDirectiveTests()
                 .AddSuggestions("apple", "banana", "cherry");
 
             _vegetableOption = new Option<string>("--vegetable")
-                .AddSuggestions(_ => new[] { "asparagus", "broccoli", "carrot" });
+                .AddSuggestions("asparagus", "broccoli", "carrot");
```

Comment:
Although it could be a lot more apparent, this is deliberately using the delegate-based approach as opposed to the `params` approach on line 25. So please revert this in case it represents a loss of scenario coverage.

Reply:
Ok. Only this one, or should I revert all my changes from delegate to `params`?